main.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-restricted-syntax */
+
+localStorage.books = JSON.stringify([]);
+
+function displayBooks() {
+  const container = document.getElementById('container');
+  container.innerHTML = '';
+
+  const books = JSON.parse(localStorage.books);
+  if (books.length === 0) {
+    container.innerHTML = 'Shelf is empty 🙄❗';
+  } else {
+    for (const book of books) {
+      const title = document.createElement('p');
+      const author = document.createElement('p');
+      const removeBtn = document.createElement('button');
+      removeBtn.setAttribute('data-id', books.indexOf(book));
+      removeBtn.innerText = 'Remove';
+      title.innerHTML = book.title;
+      author.innerHTML = book.author;
+      container.append(title, author, removeBtn);
+    }
+  }
+}
+
+function removeBook(id) {
+  const books = JSON.parse(localStorage.books);
+  books.splice(id, 1);
+  localStorage.books = JSON.stringify(books);
+  displayBooks();
+  return books;
+}
+
+function createBook(title, author) {
```

Comment:
- [ ] Entries are **not able to be persisted successfully** in the _local storage_. When the _Add book_ button is clicked after entering some data, and subsequently, the page is refreshed the entered data vanishes from the local storage. Kindly improve upon this aspect.